tests/integration/testdata/sync/infra/after/Ruby/function/app.rb

```diff
@@ -0,0 +1,22 @@
+require 'httparty'
+require 'json'
+require 'layer'
+
+def lambda_handler(event:, context:)
+  # Sample pure Lambda function that returns a message and a location
+
+  begin
+    response = HTTParty.get('http://checkip.amazonaws.com/')
```

Comment:
Same comment as the python one.

Reply:
Same as above, considering this to be a useful component of the test